src/bitcoin-cli.cpp

```diff
@@ -32,14 +32,16 @@ static const int CONTINUE_EXECUTION=-1;
 
 std::string HelpMessageCli()
 {
+    const std::unique_ptr<CBaseChainParams> defaultBaseParams(CBaseChainParams::Factory(CBaseChainParams::MAIN));
```

Comment:
Lines like this (as there are many) can avoid some redundancy by using `const auto defaultBaseParmams = ...` instead.